config/default.yaml

```diff
@@ -120,6 +120,9 @@ user:
 transcoding:
   enabled: true
   threads: 1
+  concurrency: 1
```

Comment:
Maybe add a comment explaining the difference between the `threads` configuration